metadata-ingestion/src/datahub/ingestion/source/dbt/dbt_common.py

```diff
@@ -1126,24 +1126,25 @@ def _infer_schemas_and_update_cll(  # noqa: C901
             elif node.compiled_code:
                 try:
                     # Add CTE stops based on the upstreams list.
+                    cte_mapping = {
```

Comment:
Is there any actual change here? Seems identical

Reply:
No this part is the same